### PR TITLE
Fix typos and other mistakes in the provider documentation

### PR DESCRIPTION
**Pull request by adambarreiro**
This PR fixes all found typos and other mistakes in the provider documentation.

Kudos to @dataclouder for finding most of them 👍

### CODE REVIEW

File: website/docs/r/subscribed_catalog.html.markdown [hunk 1]
```diff
@@ -62,22 +62,23 @@ The following arguments are supported:
 * `org` - (Optional) The name of organization to use, optional if defined at provider level.
 * `name` - (Required) Catalog name
 * `storage_profile_id` - (Optional) Allows to set specific storage profile to be used for catalog.
-* `delete_recursive` - (Optional) When destroying use delete_recursive=True to remove the catalog and any objects it contains that are in a state that normally allows removal.
-* `delete_force` -(Optional) When destroying use delete_force=True with delete_recursive=True to remove a catalog and any objects it contains, regardless of their state.
+* `delete_recursive` - (Optional) When destroying use `delete_recursive=true` to remove the catalog and any objects it contains that are in a state that normally allows removal.
+* `delete_force` -(Optional) When destroying use `delete_force=true` with `delete_recursive=true` to remove a catalog and any objects it contains, regardless of their state.
```
Comment (Didainius) [inline website/docs/r/subscribed_catalog.html.markdown:66]:
```suggestion
* `delete_force` - (Optional) When destroying use `delete_force=true` with `delete_recursive=true` to remove a catalog and any objects it contains, regardless of their state.
```

File: website/docs/r/subscribed_catalog.html.markdown [hunk 2]
```diff
@@ -62,22 +62,23 @@ The following arguments are supported:
...
 * `sync_on_refresh` - (Optional) Boolean value that shows if sync should be performed on every refresh.
-* `cancel_failed_tasks` - (Optional) When true, the subscribed catalog will attempt canceling failed tasks.
-* `sync_all` - (Optional) If true, synchronise this catalog and all items. 
-* `sync_catalog` (Optional) If true, synchronise this catalog. Not to be used when `sync_all` is set. This operation fetches the list of items. If `make_local_copy` is set, it also synchronises all the items.
-* `sync_all_vapp_templates` (Optional) If true, synchronise all vApp templates. Not to be used when `sync_all` is set.
-* `sync_all_media_items` (Optional) If true, synchronise all media items. Not to be used when `sync_all` is set.
+* `cancel_failed_tasks` - (Optional) When `true`, the subscribed catalog will attempt canceling failed tasks.
+* `sync_all` - (Optional) If `true`, synchronise this catalog and all items. 
+* `sync_catalog` (Optional) If `true`, synchronise this catalog. Not to be used when `sync_all` is set. This operation fetches the list of items. If `make_local_copy` is set, it also synchronises all the items.
```
Comment (Didainius) [inline website/docs/r/subscribed_catalog.html.markdown:76]:
```suggestion
* `sync_catalog` - (Optional) If `true`, synchronise this catalog. Not to be used when `sync_all` is set. This operation fetches the list of items. If `make_local_copy` is set, it also synchronises all the items.
```

File: website/docs/r/subscribed_catalog.html.markdown [hunk 3]
```diff
@@ -62,22 +62,23 @@ The following arguments are supported:
...
+* `sync_all_vapp_templates` (Optional) If `true`, synchronise all vApp templates. Not to be used when `sync_all` is set.
```
Comment (Didainius) [inline website/docs/r/subscribed_catalog.html.markdown:77]:
```suggestion
* `sync_all_vapp_templates` - (Optional) If `true`, synchronise all vApp templates. Not to be used when `sync_all` is set.
```

File: website/docs/r/subscribed_catalog.html.markdown [hunk 4]
```diff
@@ -62,22 +62,23 @@ The following arguments are supported:
...
+* `sync_all_media_items` (Optional) If `true`, synchronise all media items. Not to be used when `sync_all` is set.
```
Comment (Didainius) [inline website/docs/r/subscribed_catalog.html.markdown:78]:
```suggestion
* `sync_all_media_items` - (Optional) If `true`, synchronise all media items. Not to be used when `sync_all` is set.
```

File: website/docs/r/subscribed_catalog.html.markdown [hunk 5]
```diff
@@ -62,22 +62,23 @@ The following arguments are supported:
...
 * `sync_vapp_templates` (Optional) Synchronise a list of vApp templates. Not to be used when `sync_all` or `sync_all_vapp_templates` are set.
```
Comment (Didainius) [inline website/docs/r/subscribed_catalog.html.markdown:79]:
```suggestion
* `sync_vapp_templates` - (Optional) Synchronise a list of vApp templates. Not to be used when `sync_all` or `sync_all_vapp_templates` are set.
```

File: website/docs/r/subscribed_catalog.html.markdown [hunk 6]
```diff
@@ -62,22 +62,23 @@ The following arguments are supported:
...
 * `sync_media_items` (Optional) Synchronise a list of media items. Not to be used when `sync_all` or `sync_all_media_items` are set.
```
Comment (Didainius) [inline website/docs/r/subscribed_catalog.html.markdown:80]:
```suggestion
* `sync_media_items` - (Optional) Synchronise a list of media items. Not to be used when `sync_all` or `sync_all_media_items` are set.
```

**Review by Didainius — APPROVED** [review]
Thanks!
Noted a few missing/misaligned dashes `-`

**Review by dataclouder — APPROVED** [review]

File: website/docs/r/subscribed_catalog.html.markdown (hunk 1 above)
Comment (adambarreiro) [reply website/docs/r/subscribed_catalog.html.markdown]: Fixed!

File: website/docs/r/subscribed_catalog.html.markdown (hunk 2 above)
Comment (adambarreiro) [reply website/docs/r/subscribed_catalog.html.markdown]: Fixed!

File: website/docs/r/subscribed_catalog.html.markdown (hunk 4 above)
Comment (adambarreiro) [reply website/docs/r/subscribed_catalog.html.markdown]: Fixed!

File: website/docs/r/subscribed_catalog.html.markdown (hunk 5 above)
Comment (adambarreiro) [reply website/docs/r/subscribed_catalog.html.markdown]: Fixed!

File: website/docs/r/subscribed_catalog.html.markdown (hunk 6 above)
Comment (adambarreiro) [reply website/docs/r/subscribed_catalog.html.markdown]: Fixed!

**Pull request merged**